Sources/Commands/SwiftBuildTool.swift

```diff
@@ -213,14 +211,14 @@ public struct SwiftBuildTool {
         print("USAGE: swift build [mode] [options]")
         print("")
         print("MODES:")
-        print("  --configuration <value>        Build with configuration (debug|release) [-c]")
-        print("  --clean[=<mode>]               Delete artifacts (build|dist)")
+        print("  --configuration <value>   Build with configuration (debug|release)")
+        print("  --clean [<mode>]          Delete artifacts (build|dist)")
         print("")
         print("OPTIONS:")
         print("  --chdir <path>       Change working directory before any other operation [-C]")
         print("  --build-path <path>  Specify build directory")
         print("  --color <mode>       Specify color mode (auto|always|never)")
-        print("  -v[v]                Increase verbosity of informational output")
+        print("  --verbose            Increase verbosity of informational output [-v]")
```

Comment:
I agree — I followed precedent that was already there, but I've changed around all the cases now (in the updated PR).